.travis.yml

```diff
@@ -14,7 +14,7 @@ sudo: false  # use the new container-based Travis infrastructure
 dist: trusty # Ubuntu 14.04
 
 mono:
-  - latest
+  - nightly
```

Comment:
yeah, 4.8.2 doesn't seem to exist, and 4.8.1 fails to build (probably because of xbuild vs msbuild)

https://travis-ci.org/fsprojects/Paket/builds/260792315

but nightly looks good so far